packages/plugins/users-permissions/server/services/user.js

```diff
@@ -35,10 +35,6 @@ module.exports = ({ strapi }) => ({
    * @return {Promise}
    */
   async add(values) {
-    if (values.password) {
-      values.password = await getService('user').hashPassword(values);
-    }
-
```

Comment:
If we just remove this, the password won't be encrypted at all.
I would suggest we use the entity service here instead of doing a raw query, this way we would benefit from the hashing transformation (just like in the update).

Smth like that:
```js
return strapi.entityService.create('plugin::users-permissions.user', {
    data: values,
    populate: ['role'],
});
```

Reply:
True, my bad – missed this one!

Corrected it 👍